tests/yam/migration/setup_registration.pm

```diff
@@ -19,7 +20,10 @@ sub run {
     @filter{@addons_drop} = {};
 
     select_console('root-console');
-    assert_script_run('SUSEConnect -d');
+    foreach my $addon (@addons_drop) {
+        my $pkg_name = ($addon eq 'ltss') ? "sles-$addon-release" : "sle-module-$addon-release";
+        zypper_call("rm $pkg_name");
+    }
```

Comment:
I think it's better to do assert_script_run('SUSEConnect --debug --cleanup'); first, then remove dropped modules' release package.

Reply:
@chcao  I've changed and please take a look again, thanks